libr/util/Makefile

```diff
@@ -12,6 +12,7 @@ OBJS+=strpool.o bitmap.o strht.o p_date.o p_format.o print.o
 OBJS+=p_seven.o slist.o randomart.o log.o zip.o debruijn.o
 OBJS+=utf8.o strbuf.o lib.o name.o spaces.o
 OBJS+=diff.o bdiff.o stack.o queue.o tree.o des.o
+OBJS+=swap.o
```

Comment:
r_mem_copyendian was suposed to be renamed to r_mem_swap at some point, and this is kind of duplicating its functionalities, but its fine.